model/constants.py

```diff
@@ -286,5 +286,16 @@ class MediaTypes(object):
         ".htm" : TEXT_HTML_MEDIA_TYPE,
         ".jpeg" : JPEG_MEDIA_TYPE,
     }
+    # PYTHON3 TODO: This happens to work in Python 2, but in Python 3,
+    # items come out of MEDIA_TYPE_FOR_EXTENSION in the order they
+    # were inserted, so we need to take steps to ensure
+    # SCORM_MEDIA_TYPE doesn't claim '.zip'.
```

Comment:
Are you sure this always works in Python 2?

Reply:
I'm not, but it's pretty stable -- otherwise I feel like we'd see intermittent test failures.
Python 2.7 has a class called `OrderedDict` which we can use here to get Python 3 behavior in Python 2. I can use that here.